README.md

```diff
@@ -1,3 +1,12 @@
+PLMultiAnalyzer
+===============
+A Lucene custom analyzer that allow for indexing multiple tokens for a single term 
```

Comment:
Just some small English changes -

"with punctuations," -> "with punctuation."

"as it making Lucene doing more Exact match search" -> "as it causes Lucene to perform a more exact search"